Zupreme.xcodeproj/project.pbxproj

```diff
@@ -1310,8 +1330,10 @@
 		484B065C20BA0C2F00C4E50A /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				GCC_OPTIMIZATION_LEVEL = 0;
```

Comment:
are those build settings wanted? Needed? If so we should update the values in the `xcconfig` files. Placed in `Configurations` folder in the root and `Source/Application/Configurations`.

If you select the target and Build Settings we want "Customized" tab to be empty :)

Reply:
I added them so we can Debug properly in Xcode. We can't debug with -O (in some cases, the breakpoints are ignored and variables can't be viewed with -O). The usual Xcode config is "No optimization [-Onone]" for Debug and "Optimize for Speed [-O]" for Release.

Moving them to Configurations - Debug.

GCC_OPTIMIZATION_LEVEL is not needed since we'll never have gcc compilable code, but I could leave it as an easter egg.

Reply:
Yeah I've been meaning to set `SWIFT_OPTIMIZATION_LEVEL = -Onone`. I can take care of it in a PR soon, if that is okay? 

I actually intentionally used `-O` even for Debug builds because the execution time of some cryptographical functions are 100x slower without it. So with `Onone` it takes around 3 seconds to decrypt the `Keystore`, which is done everytime you enter the Encryption passphrase when sending. So this 100x perfomrance difference is really quite annoying. 

Instead of the quite custom build setting `SWIFT_OPTIMIZATION_LEVEL = -O` for `Debug` config that is being used now, what about we add a new Scheme, called `Zupreme-Optimized` or similar, which uses `Release` as *Build Configuration* for *Run*? Which we can use normally for development running on device, when we need to test Creation and Restoration of Wallet. And testing Sending of Zillings?